### PR TITLE
Monster Nests

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
For spawning simple animals in away missions, dungeons, lavaland, whatever.

Did them as mobs so you can have sentient/mobile nests (like a robot that spits out defense drones, or a giant hivelord that spawns more hivelords that spawn hive things)

### CODE REVIEW

**Comment by WJohn** [comment]
SLOWLY BUT SURELY, LAVA LAND WILL MAKE ITS DEBUT.

**Comment by Cheridan** [comment]
While you're at it, can you turn the existing hivebot beacon thingie into one of these?

**Comment by KorPhaeron** [comment]
@Cheridan 

Yeah sure

**Comment by Steelpoint** [comment]
Maybe now we can have the Hivebot invasion of 2555.

File: code/modules/mob/living/simple_animal/spawner.dm [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+	stop_automated_movement = 1
+	wander = 0
+	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 5, "min_n2" = 0, "max_n2" = 0)
+	minbodytemp = 0
+	maxbodytemp = 350
+
+
+/mob/living/simple_animal/hostile/spawner/Destroy()
+	for(var/mob/living/simple_animal/L in spawned_mobs)
+		if(L. == src)
```
Comment (RemieRichards) [inline code/modules/mob/living/simple_animal/spawner.dm]: ```L.``` hmmmmmm

Comment (KorPhaeron) [inline code/modules/mob/living/simple_animal/spawner.dm]: I can type

**Comment by tuypo1** [comment]
:+1:

**Comment by KorPhaeron** [comment]
@Razharas 

I wasn't done aaaa

**Pull request merged**